tests/cypress/integration/stateWorkflow/stateSubmissionForm/documents.spec.ts

```diff
@@ -9,6 +9,17 @@ describe('documents', () => {
             const draftSubmissionID = pathname.split('/')[2]
             cy.visit(`/submissions/${draftSubmissionID}/documents`)
 
+            // Add two of the same document
+            cy.visit(`/submissions/${draftSubmissionID}/documents`)
+            cy.findByTestId('file-input-input').attachFile([
+                'documents/trussel-guide.pdf'
+            ])
+            cy.findByTestId('file-input-input').attachFile(
```

Comment:
For the future us - This is a cypress test to ensure said weird bug doesnt recurr.